openmicroscopy/ome-files-performance
# Add JACE tests

Duplicates of the existing C++ and Java tests, using JACE.

I've had to import some of the generated bits from bio-formats-jace due to it not packaging them.

Testing:

- build bio-formats-jace
- run cmake, with CMAKE_PREFIX_PATH including the bio-formats-jace dist/bio-formats-jace directory; it should autodetect bio-formats-jace
- build
- copy the bio-formats-jace `jar` directory into the build directory (maybe could be added to `CLASSPATH`?)
- run the examples as for the existing C++ tests
